judge/views/organization.py

```diff
@@ -301,13 +302,17 @@ def form_valid(self, form):
             # slug is show in url
             # short_name is show in ranking
             org.short_name = org.slug[:20]
-            org.admins.add(self.request.user.profile)
             org.save()
+            all_admins = org.admins.all()
+            g = Group.objects.get(name='Org Admin')
```

Comment:
move this to `setting` file instead of hardcode. Like `GROUP_PERMISSION_FOR_ORG_ADMIN`